### DISCUSSION TITLE
《GitHub 社区指导方针》选编——维护强大的社区

### DISCUSSION THREAD

**Discussion by Silverteal** [Meloong-Git/PCL#4339] [讨论]
鉴于最近一些社群成员的不当评论方式已经引发了一系列不愉快事件，我感此必要摘录此文的一部分，以此为鉴。

原载于 [Github 社区指导方针](https://docs.github.com/zh/site-policy/github-terms/github-community-guidelines)

---
## 维护强大的社区
GitHub 社区的主要目的是协作处理软件项目。 我们致力于维持一个用户能够自由表达意见并对彼此想法（包括技术和其他方面）提出挑战的社区。 同时，重要的是用户要保持尊重，并为其他人留出公开贡献的空间。 为了营造一个安全和富有成效的环境，我们鼓励社区成员参考这些准则，以了解如何在我们的平台上进行互动。 

* **热情和开放** - 每天都有新用户加入我们的社区。 有些是成熟的开发人员，而另一些则刚刚入门。 请对有其他想法和经验水平的用户持开放态度。 善于接纳其他人的意见，友善对待新的协作者和刚入门的协作者。

* **尊重他人** - 在协作环境中工作难免会发生分歧。 但请记住，您的批评要对事不对人。 分享深思熟虑的建设性批评，并对与您互动的人保持礼貌。 如果您无法以尊重的方式参与，请考虑做出一些让步或使用我们的一些缓和工具来缓解紧张的氛围。

* **富有同情心** - GitHub 是一个全球性社区，社区中的用户背景各异，他们的观点也各不相同，其中许多人的背景和观点可能与您不同。 试着设身处地为他人着想，在对他们说话之前，先了解他们的感受。 尽最大努力帮助使 GitHub 成为一个任何人都能放心地作出贡献、参与讨论和分享不同想法的社区。

**Comment by wuliaodexiaoluo** [comment]
> [Github 社区指导方针](https://docs.github.com/zh/site-policy/github-terms/github-community-guidelines)

纠正:是 GitHub 而不是 Github
